src/components/PDFView/index.js

```diff
@@ -17,22 +22,57 @@ const defaultProps = {
     style: {},
 };
 
-/**
- * On web, we use a WebView pointed to a pdf renderer
- *
- * @param props
- * @returns {JSX.Element}
- */
+class PDFView extends PureComponent {
+    constructor(props) {
+        super(props);
+        this.state = {
+            numPages: null,
+        };
+        this.onDocumentLoadSuccess = this.onDocumentLoadSuccess.bind(this);
+        console.debug(props);
+    }
+
+    onDocumentLoadSuccess = ({numPages}) => {
```

Comment:
Please avoid using class property syntax. We don't use it anywhere else.
And make sure to document all methods with JSDocs.

Reply:
ok